tests/subsys/fs/fat_fs_api/src/test_fat_file.c

```diff
@@ -38,7 +38,7 @@ static int test_file_open(void)
 		TC_PRINT("File now exists %s\n", TEST_FILE);
 	}
 
-	return res;
```

Comment:
Are these changes necessary?  I agree this source file is wrong in not explictly returning the `TC_PASS` it looks for, but if this needs to be done it should be in a separate cleanup commit, and all tests should be updated to do this (and the one that's not `static` made so).

Reply:
Changes removed.  Will fix in other PR.